docs/security-champion/7-about.md

```diff
@@ -0,0 +1,29 @@
+# About the network
+
+The Security Champion Network is intended to be a community for Security Champions in Equinor. Software development over the last years has rapidly evolved from big development teams consisting of dozens of developers to smaller autonomous teams where we are today. With greater responsibility of the whole lifecycle of applications, modern DevOps teams are also expected to handle security.
+
+This network rises from the realization that this shift-left thinking when it comes to security requires competence building within the developer teams. As the network grows we strive for it to become a powerhouse in Equinor when it comes to application security. It will be a place where teams can exchange experiences - _both good and bad_, and hopefully learn from each others.
+
+## Desired outcome
+
+Create a lively community for people working with Application Security in Equinor. Knowledge is shared across teams, and we are then able to scale security in a more impactful way.
+
+## Activities
+
+Please check our [event](./events/index.md) site for info about upcoming and past Security Champion network events.
+
+The main communication channel for the community is [Slack](https://app.slack.com/client/T02JL00JU/CMM6FSW5V). This is where people can post questions exchange experiences when it comes to different tools and technologies etc.
+
+## Responsibilities
+
+Everyone in the community is responsible for keeping these info sites up to date. They are all based on markdown, don't hesitate to make a PR with fixes, or some new guidelines!
```

Comment:
Is this really everyone responsibility? I'd twist this towards "contributions are welcome" rather than "you are here, now it's your responsibility"

Reply:
Or just drop this paragraph, and keep only the following one